gemfiles/activerecord-5.0/Gemfile.base

```diff
@@ -1,3 +1,3 @@
 eval File.read File.expand_path('../../Gemfile.base', __FILE__)
 
-gem "activerecord", "~> 4.2.6"
+gem "activerecord", ">= 5.0.0.beta1", "< 5.1"
```

Comment:
oh duh of course it's generated by `schema_dev`.  Now that AR 5.0 is out of beta i've switched that to `~> 5.0.0` in `schema_dev` and pushed a new release